content/manuals/billing/3d-secure.md

```diff
@@ -2,6 +2,7 @@
 title: 3D Secure authentication
 description: Learn about 3D Secure support for Docker billing.
 keywords: billing, renewal, payments, subscriptions
+weight: 500
```

Comment:
I've used increments of 10 in most other cases, starting from 10, 20, 30, and so on. Any reason we start at 300 for this section?